addon/lib/user.js

```diff
@@ -1,7 +1,11 @@
+const { Cc, Ci, Cu } = require('chrome');
```

Comment:
I think jshint gives unused import warnings for some of these.  Not sure how I feel – this is a common import, but I also like to keep things jshint clean even when it makes me do things that feel nitpicky.